test/kaldi_compatibility_impl.py

```diff
@@ -61,7 +55,7 @@ def assert_equal(self, output, *, expected, rtol=None, atol=None):
         expected = expected.to(dtype=self.dtype, device=self.device)
         self.assertEqual(output, expected, rtol=rtol, atol=atol)
 
-    @unittest.skipIf(_not_available('apply-cmvn-sliding'), '`apply-cmvn-sliding` not available')
+    @common_utils.skipIfNoExec('apply-cmvn-sliding')
```

Comment:
nit: again, i'm not sure why this is changing as part of this pr?

Reply:
So as to avoid two duplicated implementation. 

`kaldi_compatibility_impl` had this original version of `skipIfNoExec` implementation (which is `_not_available` func + `unittest.skipIf`), and now that implementation is promoted to common utility because the new `sox_io` test uses it too.